### ISSUE TITLE
PLY File segfaults

### ISSUE THREAD

**Issue by mwestphal** [f3d-app/f3d#618]
**Describe the bug**
A specific PLY file segfaults

**To Reproduce**
 0. [cube_quads.ply.zip](https://github.com/f3d-app/f3d/files/10611563/cube_quads.ply.zip)

 1. Open the file using `f3d --dry-run cube_quads.ply`
 2. step 2

```

ply_get_element: unexpected end of file
f3d: /home/glow/dev/vtk/vtk1/src/IO/PLY/vtkPLY.cxx:1566: static bool vtkPLY::ascii_get_element(PlyFile*, char*): Assertion `0' failed.

Program received signal SIGABRT, Aborted.
0x00007ffff785e64c in ?? () from /usr/lib/libc.so.6
(gdb) bt
#0  0x00007ffff785e64c in ?? () from /usr/lib/libc.so.6
#1  0x00007ffff780e938 in raise () from /usr/lib/libc.so.6
#2  0x00007ffff77f853d in abort () from /usr/lib/libc.so.6
#3  0x00007ffff77f845c in ?? () from /usr/lib/libc.so.6
#4  0x00007ffff7807486 in __assert_fail () from /usr/lib/libc.so.6
#5  0x00007ffff64ed76b in vtkPLY::ascii_get_element (plyfile=0x5555560bd790, elem_ptr=0x7fffffffd010 "\020w\vVU\004")
    at /home/glow/dev/vtk/vtk1/src/IO/PLY/vtkPLY.cxx:1566
#6  0x00007ffff64ec717 in vtkPLY::ply_get_element (plyfile=0x5555560bd790, elem_ptr=0x7fffffffd010) at /home/glow/dev/vtk/vtk1/src/IO/PLY/vtkPLY.cxx:1021
#7  0x00007ffff64f3f2c in vtkPLYReader::RequestData (this=0x555555fd0260, outputVector=0x555555fd1950)
    at /home/glow/dev/vtk/vtk1/src/IO/PLY/vtkPLYReader.cxx:488
#8  0x00007ffff2339f08 in vtkPolyDataAlgorithm::ProcessRequest (this=0x555555fd0260, request=0x555555fd22e0, inputVector=0x0, outputVector=0x555555fd1950)
    at /home/glow/dev/vtk/vtk1/src/Common/ExecutionModel/vtkPolyDataAlgorithm.cxx:88
#9  0x00007ffff230ecbb in vtkExecutive::CallAlgorithm (this=0x555555fd1820, request=0x555555fd22e0, direction=1, inInfo=0x0, outInfo=0x555555fd1950)
    at /home/glow/dev/vtk/vtk1/src/Common/ExecutionModel/vtkExecutive.cxx:735
#10 0x00007ffff2305048 in vtkDemandDrivenPipeline::ExecuteData (this=0x555555fd1820, request=0x555555fd22e0, inInfo=0x0, outInfo=0x555555fd1950)
    at /home/glow/dev/vtk/vtk1/src/Common/ExecutionModel/vtkDemandDrivenPipeline.cxx:462
#11 0x00007ffff22f8281 in vtkCompositeDataPipeline::ExecuteData (this=0x555555fd1820, request=0x555555fd22e0, inInfoVec=0x0, outInfoVec=0x555555fd1950)
    at /home/glow/dev/vtk/vtk1/src/Common/ExecutionModel/vtkCompositeDataPipeline.cxx:163
#12 0x00007ffff23045f5 in vtkDemandDrivenPipeline::ProcessRequest (this=0x555555fd1820, request=0x555555fd22e0, inInfoVec=0x0, outInfoVec=0x555555fd1950)
    at /home/glow/dev/vtk/vtk1/src/Common/ExecutionModel/vtkDemandDrivenPipeline.cxx:261
#13 0x00007ffff23d0534 in vtkStreamingDemandDrivenPipeline::ProcessRequest (this=0x555555fd1820, request=0x555555fd22e0, inInfoVec=0x0, 
    outInfoVec=0x555555fd1950) at /home/glow/dev/vtk/vtk1/src/Common/ExecutionModel/vtkStreamingDemandDrivenPipeline.cxx:344
#14 0x00007ffff2304e2d in vtkDemandDrivenPipeline::UpdateData (this=0x555555fd1820, outputPort=0)
    at /home/glow/dev/vtk/vtk1/src/Common/ExecutionModel/vtkDemandDrivenPipeline.cxx:419
#15 0x00007ffff23d08c4 in vtkStreamingDemandDrivenPipeline::Update (this=0x555555fd1820, port=0, requests=0x0)
    at /home/glow/dev/vtk/vtk1/src/Common/ExecutionModel/vtkStreamingDemandDrivenPipeline.cxx:418
#16 0x00007ffff23d074d in vtkStreamingDemandDrivenPipeline::Update (this=0x555555fd1820, port=0)
    at /home/glow/dev/vtk/vtk1/src/Common/ExecutionModel/vtkStreamingDemandDrivenPipeline.cxx:381
#17 0x00007ffff22ed38a in vtkAlgorithm::Update (this=0x555555fd0260, port=0) at /home/glow/dev/vtk/vtk1/src/Common/ExecutionModel/vtkAlgorithm.cxx:1492
#18 0x00007ffff22ed355 in vtkAlgorithm::Update (this=0x555555fd0260) at /home/glow/dev/vtk/vtk1/src/Common/ExecutionModel/vtkAlgorithm.cxx:1486
#19 0x00007ffff7e963b4 in vtkF3DGenericImporter::ImportBegin (this=0x5555556edae0)
    at /home/glow/dev/f3d/f3d/src/library/VTKExtensions/Readers/vtkF3DGenericImporter.cxx:345
#20 0x00007ffff5263a9f in vtkImporter::Read (this=0x5555556edae0) at /home/glow/dev/vtk/vtk1/src/IO/Import/vtkImporter.cxx:85
#21 0x00007ffff7e78198 in vtkImporter::Update (this=0x5555556edae0) at /home/glow/dev/vtk/vtk1/src/IO/Import/vtkImporter.h:92
#22 0x00007ffff7e77cb9 in f3d::detail::loader_impl::loadFile (this=0x55555570cdd0, load=f3d::loader::LoadFileEnum::LOAD_CURRENT)
    at /home/glow/dev/f3d/f3d/src/library/src/loader_impl.cxx:379
#23 0x000055555561bd8d in F3DStarter::LoadFile (this=0x7fffffffe730, load=f3d::loader::LoadFileEnum::LOAD_CURRENT)
    at /home/glow/dev/f3d/f3d/src/application/F3DStarter.cxx:357
#24 0x000055555561b038 in F3DStarter::Start (this=0x7fffffffe730, argc=3, argv=0x7fffffffe878) at /home/glow/dev/f3d/f3d/src/application/F3DStarter.cxx:202
#25 0x000055555561f4e9 in main (argc=3, argv=0x7fffffffe878) at /home/glow/dev/f3d/f3d/src/application/main.cxx:13
(gdb) q

```

**Expected behavior**
No segfault, file works correctly in blender

**Comment by mwestphal** [comment]
sister VTK issue: https://gitlab.kitware.com/vtk/vtk/-/issues/18807

**Comment by mwestphal** [comment]
Fixed in VTK, Fixed in F3D 2.1.0 binary release
